### PR TITLE
lxqt.{pcmanfm-qt,lximage-qt}: add qtimageformats plugin

### PR DESCRIPTION
**Pull request by tpwrules**
###### Description of changes
This adds the qtimageformats plugin to the LXQt file manager and image viewer for additional viewing and thumbnailing format support, including TIFF and WEBP. The closure size increase is negligible (by 650KiB to 530MiB).

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [X] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
@ofborg eval

File: pkgs/desktops/lxqt/lximage-qt/default.nix [hunk 1]
```diff
@@ -36,6 +37,7 @@ mkDerivation rec {
     qttools
     qtx11extras
     qtsvg
+    qtimageformats
```
Comment (romildo) [inline pkgs/desktops/lxqt/lximage-qt/default.nix:40]:
```suggestion
    qtimageformats # add-on module to support more image file formats
```

File: pkgs/desktops/lxqt/pcmanfm-qt/default.nix [hunk 2]
```diff
@@ -34,6 +35,7 @@ mkDerivation rec {
     qtbase
     qttools
     qtx11extras
+    qtimageformats
```
Comment (romildo) [inline pkgs/desktops/lxqt/pcmanfm-qt/default.nix:38]:
```suggestion
    qtimageformats # add-on module to support more image file formats
```

**Review by romildo — CHANGES_REQUESTED** [review]
Adding a small comment in the source, as `qtimageformats` is not required and is not listed as a dependence by upstream. Helps to remember why it is being added.

**Pull request merged**